From 5b12331cd7d268ce9681f3a488dc7745b5dc423b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:03:26 -0400
Subject: [usb] wusbcore: fix up coding style issues in wa-nep.c

Message-id: <1410887463-20674-135-git-send-email-dzickus@redhat.com>
Patchwork-id: 95209
O-Subject: [RHEL7 PATCH 134/591] USB: wusbcore: fix up coding style issues in wa-nep.c
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 2627cb08129d7fa152f97dd0732736fb2df516ee
Author: Rahul Bedarkar <rahulbedarkar89@gmail.com>
Date:   Sat Jan 4 14:11:01 2014 +0530

    USB: wusbcore: fix up coding style issues in wa-nep.c

    Signed-off-by: Rahul Bedarkar <rahulbedarkar89@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/wusbcore/wa-nep.c b/drivers/usb/wusbcore/wa-nep.c
index ada4e08..60a10d2 100644
--- a/drivers/usb/wusbcore/wa-nep.c
+++ b/drivers/usb/wusbcore/wa-nep.c
@@ -69,8 +69,8 @@ struct wa_notif_work {
  * [the wuswad daemon, basically]
  *
  * @_nw: Pointer to a descriptor which has the pointer to the
- *   @wa, the size of the buffer and the work queue
- *   structure (so we can free all when done).
+ *  @wa, the size of the buffer and the work queue
+ *  structure (so we can free all when done).
  * @returns     0 if ok, < 0 errno code on error.
  *
  * All notifications follow the same format; they need to start with a
@@ -93,7 +93,8 @@ static void wa_notif_dispatch(struct work_struct *ws)
 {
  void *itr;
  u8 missing = 0;
- struct wa_notif_work *nw = container_of(ws, struct wa_notif_work, work);
+ struct wa_notif_work *nw = container_of(ws, struct wa_notif_work,
+      work);
  struct wahc *wa = nw->wa;
  struct wa_notif_hdr *notif_hdr;
  size_t size;
@@ -271,7 +272,8 @@ int wa_nep_create(struct wahc *wa, struct usb_interface *iface)
  wa->nep_buffer_size = 1024;
  wa->nep_buffer = kmalloc(wa->nep_buffer_size, GFP_KERNEL);
  if (wa->nep_buffer == NULL) {
-  dev_err(dev, "Unable to allocate notification's read buffer\n");
+  dev_err(dev,
+   "Unable to allocate notification's read buffer\n");
   goto error_nep_buffer;
  }
  wa->nep_urb = usb_alloc_urb(0, GFP_KERNEL);
-- 
1.7.1